R/checkers.R

```diff
@@ -0,0 +1,148 @@
+check_required_inputs_provided <- function(cases,
+                                           reference_date,
+                                           group,
+                                           call = rlang::caller_env()) {
+  rlang::check_required(cases, "cases")
```

Comment:
`rlang` is a good choice here but could also use `checkmate`?

Reply:
You use `checkmate` in epinowcast?

Reply:
no but we probably should. its used a lot in `EpiNow2`. In `epinowcast` we use a hand rolled `data.table` checker.

Reply:
I like the control of the rlang builtins. Your experience on EpiNow2 was that the convenience was worth it?

Reply:
I'm going to leave it with `{rlang}` here, but I like the idea in general.